server/runtime/src/test/java/org/infinispan/server/test/InfinispanServerDriver.java

```diff
@@ -83,6 +84,9 @@ public void prepare(String name) {
          throw new RuntimeException("Failed to create server configuration directory " + confDir);
       }
       URL configurationFileURL = getClass().getClassLoader().getResource(configuration.configurationFile());
+      if (new File(configuration.configurationFile()).isAbsolute()) {
+         configurationFileURL = Exceptions.unchecked(() -> new File(configuration.configurationFile()).toURI().toURL());
```

Comment:
IMO the declaration of `configurationFileURL` should not initialize it, and an `else` branch should initialize it instead.

Reply:
> IMO the declaration of `configurationFileURL` should not initialize it, and an `else` branch should initialize it instead.

On this way I will need a third `if` to check if configurationFileURL is null